sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py

```diff
@@ -0,0 +1,878 @@
+        :type tags: dict(str, str)
+        :returns: The updated Certificate
+        :rtype: ~azure.security.keyvault.certificates._models.Certificate
+        :raises:
+         :class:`KeyVaultErrorException<azure.keyvault.v7_0.models.KeyVaultErrorException>`
+        """
+        if enabled is not None or not_before is not None or expires is not None:
+            attributes = self._client.models.CertificateAttributes(
+                enabled=enabled, not_before=not_before, expires=expires
+            )
+        else:
+            attributes = None
+
+        bundle = self._client.update_certificate(
+            vault_base_url=self.vault_url,
+            certificate_name=name,
+            certificate_version=version or "",
+            certificate_attributes=attributes,
+            tags=tags,
+            **kwargs
+        )
+        return Certificate._from_certificate_bundle(certificate_bundle=bundle)
+
+    @distributed_trace
+    def backup_certificate(self, name, **kwargs):
+        # type: (str) -> bytes
+        """Backs up the specified certificate.
+
+        Requests that a backup of the specified certificate be downloaded
+        to the client. All versions of the certificate will be downloaded.
+        This operation requires the certificates/backup permission.
+
+        :param name: The name of the certificate.
+        :type name: str
+        :return: the backup blob containing the backed up certificate.
+        :rtype: bytes
+        :raises:
+         :class:`KeyVaultErrorException<azure.keyvault.v7_0.models.KeyVaultErrorException>`
+        """
+        backup_result = self._client.backup_certificate(
+            vault_base_url=self.vault_url,
+            certificate_name=name,
+            error_map={404: ResourceNotFoundError},
+            **kwargs
+        )
+        return backup_result.value
+
+    @distributed_trace
+    def restore_certificate(self, backup, **kwargs):
+        # type: (bytes) -> Certificate
+        """Restores a backed up certificate to a vault.
+
+        Restores a backed up certificate, and all its versions, to a vault.
+        this operation requires the certificates/restore permission.
+
+        :param backup: The backup blob associated with a certificate bundle.
+        :type backup bytes
+        :return: The restored Certificate
+        :rtype: ~azure.security.keyvault.certificates._models.Certificate
+        :raises:
+         :class:`KeyVaultErrorException<azure.keyvault.v7_0.models.KeyVaultErrorException>`
+        """
+        bundle = self._client.restore_certificate(
+            vault_base_url=self.vault_url,
+            certificate_bundle_backup=backup,
+            error_map={409: ResourceExistsError},
+            **kwargs
+        )
+        return Certificate._from_certificate_bundle(certificate_bundle=bundle)
+
+    @distributed_trace
+    def list_deleted_certificates(self, include_pending=None, **kwargs):
```

Comment:
We have an improved paging object available for list methods, take a look at #6558 to see how it's used. (An update to the generated code is required, so you'll also need an updated copy of `_shared`.)